lib/eventemitter2.js

```diff
@@ -570,23 +960,44 @@
 
     this._events || init.call(this);
 
-    var type = arguments[0];
+    var type = arguments[0], wildcard= this.wildcard, ns, kind, containsSymbol;
+    var args,l,i,j;
 
     if (type === 'newListener' && !this._newListener) {
         if (!this._events.newListener) { return Promise.resolve([false]); }
     }
 
+    if (wildcard) {
```

Comment:
How much does this symbol stuff affect performance BTW? Do you have benchmarks pre and post this feature?

Reply:
I don’t think there might be performance issues here due symbols. 
If symbol is supported Reflect.ownKeys or pollyfill for Reflect.ownKeys is used to enumerate the props of _events|tree objects, else if symbol is not supported it uses the pevious code logic-  only string keys & Object.keys().
So, we can loss a bit of performance only in one case- if Symbols are supported and Reflect.ownKeys is not.
I don't think that Reflect.ownKeys is slower than Object.keys.

Our benchmark only tests one simple case- on/emit/removeAllListeners, so it shows the same performance.

**MASTER**

Platform: win32, x64, 15267MB
Node version: v13.11.0
CPU: 4 x AMD Ryzen 3 2200U with Radeon Vega Mobile Gfx @ 2495MHz
----------------------------------------------------------------
EventEmitterHeatUp x 3,326,830 ops/sec ±3.57% (64 runs sampled)
EventEmitter x 3,350,339 ops/sec ±4.46% (65 runs sampled)
**EventEmitter2 x 12,590,264 ops/sec ±5.95% (64 runs sampled)**
EventEmitter2 (wild) x 5,368,343 ops/sec ±5.72% (61 runs sampled)
EventEmitter3 x 10,471,947 ops/sec ±5.33% (61 runs sampled)

**DEV**

Platform: win32, x64, 15267MB
Node version: v13.11.0
CPU: 4 x AMD Ryzen 3 2200U with Radeon Vega Mobile Gfx @ 2495MHz
----------------------------------------------------------------
EventEmitterHeatUp x 3,155,632 ops/sec ±3.90% (62 runs sampled)
EventEmitter x 3,943,529 ops/sec ±2.96% (77 runs sampled)
**EventEmitter2 x 12,985,085 ops/sec ±4.31% (67 runs sampled)**
EventEmitter2 (wild) x 4,900,476 ops/sec ±4.27% (58 runs sampled)
EventEmitter3 x 9,934,684 ops/sec ±3.57% (67 runs sampled)

Fastest is EventEmitter2